src/components/productrecommendations.main.jsx

```diff
@@ -99,58 +101,62 @@ class ProductRecommendationsDisplayMain extends React.Component {
 
   render() {
     const data = [];
-
+    const { productData } = this.state;
     /* Copy this section to add Carousel to the view */
-    if (this.state.productData._recommendations[0]._crosssell[0].links.length > 0) {
-      var product = this.state.productData._recommendations[0]._crosssell[0].links;
-      var length = this.state.productData._recommendations[0]._crosssell[0].links.length;
-      console.log(this.state.productData._recommendations[0]._crosssell[0]);
-      data.push(<div key={`_${Math.random().toString(36).substr(2, 9)}`}>
-        <label className="control-label">
-          Product Recommendations
-        </label>
-        <div className="col-md-12">
-          <div className="carousel slide" data-ride="carousel" id="theCarousel">
-            <div className="container">
-              <div className="carousel-inner">
-                {this.renderProducts(product, length, 2)}
+    if (productData._recommendations[0]._crosssell[0].links.length > 0) {
+      const product = productData._recommendations[0]._crosssell[0].links;
+      const { length } = product.length;
+      console.log(productData._recommendations[0]._crosssell[0]);// eslint-disable-line no-console
+      data.push(
+        <div key={`_${Math.random().toString(36).substr(2, 9)}`}>
```

Comment:
is there anything in productData that you can use for a key?

Reply:
Each child in an array or iterator should have a unique "key" prop. Its a requirement from react

Reply:
@jatinbehlEP Yep I know, but is there a unique identifier returned in the productData that you can use for a key rather than generating a random number? If there is, try to use that. We're using random number generation for cases where there's absolutely no unique ID returned